app/Models/User.php

```diff
@@ -40,4 +51,51 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function isLibrarian()
+    {
+        return $this->role === self::LIBRARIAN_ROLE;
+    }
+
+    public function Role()
+    {
+        if($this->role === self::LIBRARIAN_ROLE)
```

Comment:
Why not just returning the `$this->role`? Returning a hard coded string from here is ruining the purpose of creating the Constant.